repo.yml

```diff
@@ -0,0 +1,6 @@
+type: electron 
+sentry:
+    org: balenaetcher
+    team: resinio
```

Comment:
Not `balenaio` ? :wink:

Reply:
That's the sentry team the projects are accessible to

Reply:
That's a sentry internal config we haven't changed yet